src/components/radio-group/radio-group.component.ts

```diff
@@ -122,7 +122,13 @@ export default class SlRadioGroup extends ShoelaceElement implements ShoelaceFor
   }
 
   private getAllRadios() {
-    return [...this.querySelectorAll<SlRadio | SlRadioButton>('sl-radio, sl-radio-button')];
+    const slot = this.querySelector('slot');
+
+    if (slot) {
+      return slot.assignedElements({ flatten: true }) as SlRadio[] | SlRadioButton[];
+    } else {
+      return [...this.querySelectorAll<SlRadio | SlRadioButton>('sl-radio, sl-radio-button')];
+    }
```

Comment:
This is fairly brittle and will miss things like:

<slot>
   <div>
     <sl-radio />
   </div>
</slot>
```

```suggestion
// Support things that conform to an interface and explicitly marked.
return [...this.querySelectorAll<SlRadio | SlRadioButton>('sl-radio, sl-radio-button, [data-sl-radio], [data-sl-radio-button]')];
```